include/vast/Dialect/LowLevel/LowLevelOps.td

```diff
@@ -39,4 +41,138 @@ def InitializeVar
         to constructors.
     }];
 }
+
+def Br
```

Comment:
I did not have time to check yet, but I think we overlap in some parts here with core/high-level dialects.

For example we may want to have just single scoping primitives, e.g., scope end as `yield` in code?

Reply:
Will open an issue and merge this in separate PR.